src/Parser/Plural.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+/**
+ * A gettext Plural-Forms parser.
+ */
```

Comment:
This is the class description, is it possible to let it just above class definition?